myapp/db/migrate/20240606040900_add_constraints_to_tasks.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddConstraintsToTasks < ActiveRecord::Migration[6.0]
+  def change
+    change_column :tasks, :title, :string, limit: 50
+    change_column :tasks, :description, :text, limit: 500
```

Comment:
FYI: https://qiita.com/kikai8492/items/ff92b497d001f3f59a97

Reply:
I found that the reason why there doesn't work was that description was text instead of string, so the number of words could not be limited, so I modified it